internal/backend/fleetdb/consts.go

```diff
@@ -0,0 +1,26 @@
+package fleetdb
+
+// TODO: move these consts into the hollow-toolbox to share between controllers.
+
+const (
```

Comment:
some of the constants can be found here https://github.com/metal-toolbox/rivets/blob/main/fleetdb/consts.go

Reply:
Thanks and removed duplicates.